src/main/java/com/minecolonies/coremod/commands/NavigationMenu.java

```diff
@@ -26,4 +26,49 @@ public NavigationMenuType getMenuType()
     {
         return childrenMenuList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "childrenMenuList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)